cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/workers/MessagesWorker.java

```diff
@@ -107,7 +107,7 @@ private static void updateMessageWriteMetrics(MessageBatchEntity entity, BookId
 	{
 		MessagesIteratorProvider provider =
 				new MessagesIteratorProvider("get messages filtered by " + filter, filter,
-						ops.getOperators(book.getId()), book, composingService, selectQueryExecutor,
+						ops.getOperators(book.getId(), composeReadAttrs(filter.getFetchParameters())), book, composingService, selectQueryExecutor,
```

Comment:
`getOperators()` shouldn't be that customizable to define `readAttrs` each time. It returns operators for given book. And should use common `readAttrs` to obtain book info.

Reply:
Resolved in [953ac12](https://github.com/th2-net/cradleapi/pull/85/commits/953ac12318f4b7cfb6d71a0464cf462b6c661cc6)